lms/djangoapps/grades/api/v1/gradebook_views.py

```diff
@@ -771,3 +780,16 @@ def _clean_override_data(self, override_data):
             if field in allowed_fields:
                 stripped_data[field] = override_data[field]
         return stripped_data
+
+    def _log_update_result(self, request_user, user_id, usage_id, subsection_grade_model,
```

Comment:
Our convention is to break this up into one parameter per line, as described here: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html

Also, you can give these parameters default values, so you don't have to worry about passing `None` in for the model and override for the exceptional cases.  To do that, put `subsection_grade_model=None` in the method signature.

Reply:
Also, since nothing in this method body needs `self`, you can make it a static method: https://docs.python.org/2/library/functions.html#staticmethod